problem-18.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+// Takes 10 inputs from user and stores them in array and prints them
+
+int main()
+{
+    int numbers[10];
+
+    for (int i = 0; i < 10; i++)
+    {
+        printf("Enter element %d: ", i + 1);

```

Comment:
If you are specifically dealing with numbers then kindly change this statement.

Reply:
Done